includes/class-wc-siftscience-admin.php

```diff
@@ -331,7 +339,7 @@ private function get_drop_down( $id, $title, $desc, $options ) {
 				'desc' => $desc,
 				'desc_tip' => true,
 				'options' => $options,
-				'type' => 'select',
+				'type' => 'select'
```

Comment:
Actually, let's leave the comma on the last field. This is a coding style preference to make adding and removing lines easier.

Reply:
actually these methods well be deleted for now don't merge